packages/repl/src/lib/Bundler.js

```diff
@@ -13,7 +13,7 @@ export default class Bundler {
 		const hash = `${packages_url}:${svelte_url}`;
 
 		if (!workers.has(hash)) {
-			const worker = new Worker();
+			const worker = new Worker(bundler_worker, { type: 'module' });
```

Comment:
I think module workers still doesn't work on Firefox

Reply:
https://caniuse.com/?search=worker%20module

Reply:
The first result is only for `Worker` if you scroll to the last result `ES Modules in workers` you can see that Firefox doesn't have support yet

Reply:
That's for service worker